code/controllers/subsystem/mapping.dm

```diff
@@ -30,6 +30,30 @@ SUBSYSTEM_DEF(mapping)
 	seedRuins(list(level_name_to_num(MINING)), config.lavaland_budget, /area/lavaland/surface/outdoors/unexplored, GLOB.lava_ruins_templates)
 	spawn_rivers(list(level_name_to_num(MINING)))
 
+	// Now we make a list of areas for teleport locs
+	// TOOD: Make these locs into lists on the SS itself, not globs
+	for(var/area/AR in world)
+		if(AR.no_teleportlocs)
+			continue
+		if(GLOB.teleportlocs.Find(AR.name))
+			continue
+		var/turf/picked = safepick(get_area_turfs(AR.type))
+		if(picked && is_station_level(picked.z))
+			GLOB.teleportlocs += AR.name
+			GLOB.teleportlocs[AR.name] = AR
+
+	GLOB.teleportlocs = sortAssoc(GLOB.teleportlocs)
+
+	for(var/area/AR in world)
+		if(GLOB.ghostteleportlocs.Find(AR.name))
```

Comment:
Can just do `GLOB.ghostteleportlocs[AR.name]` since it's an assoc list.

Reply:
Line 52: `GLOB.ghostteleportlocs += AR.name`

Not associative

Reply:
Line 53. Assoc. It just adds the item to the list. Then at that point it uses it as assoc.
It is one way of making byond work with assoc lists.